docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -147,16 +222,151 @@ See the https://playwright.dev/docs/api/class-page#page-get-attribute[`page.getA
 <2> Use the assertion library provided by the Synthetics agent to look for the
 expected value. See https://jestjs.io/docs/expect[Jest expect docs] for more information.
 
-[[synthetics-create-test-script-recorder]]
-[NOTE]
-====
-If you want to generate code by interacting with a web page directly, you can use the *Synthetics Recorder*.
+[discrete]
+[[synthetics-assertions-methods]]
+== Methods
```

Comment:
We use the expect library from the Jest framework which itself is a standalone package. We could also point users to the website - https://jestjs.io/docs/expect. TLDR: Playwright used the same library before and recently enhanced it to support custom assertions which we clearly dont support leading to the whole confusion. 

One more idea would be to actually include only the subset of methods that we support and link the docs directly to the expect library instead. 

Thoughts?

Reply:
>We could also point users to the website - https://jestjs.io/docs/expect. 

At first glance it looks like not all the methods listed in the Jest docs are included in [`expect.ts`](https://github.com/elastic/synthetics/blob/main/src/core/expect.ts). Does that mean not all the Jest methods are supported in Synthetics? If so, could linking to the Jest site just cause the same kind of confusion we're seeing with linking to Playwright assertions? 

>One more idea would be to actually include only the subset of methods that we support and link the docs directly to the expect library instead.

When you say "link the docs directly to the expect library" do you mean the Jest site or somewhere else?

Reply:
> If so, could linking to the Jest site just cause the same kind of confusion we're seeing with linking to Playwright assertions?

Yeah thats a possibility, We didnt expose that as some of the matchers in expect does not make sense for Synthetics. 

> When you say "link the docs directly to the expect library" do you mean the Jest site or somewhere else?

Yeah directly to the jest site for each supported methods. 

I don't have any preference on whether we should add them here vs referencing the users directly there and clearly documentating that we support only partial of them. Thoughts @paulb-elastic @andrewvc

Reply:
>I don't have any preference on whether we should add them here vs referencing the users directly there and clearly documentating that we support only partial of them.

Curious what Paul and Andrew think, but I'm thinking a full list of the `expect` methods that are supported in Synthetics where each list item is just a link to the relevant part of the Jest docs makes sense to me. That seems like a good compromise -- not too much content that might become difficult to maintain, but also clarity on what is/isn't supported.

Reply:
@colleenmcginnis I 100% agree, I think this is a good compromise.
I am concerned just linking won't be clear exactly what is and isn't supported, or expected to work.  I can see this being a useful reference for us/Support as well as for users.